sdk/data/aztables/README.md

```diff
@@ -1,151 +1,342 @@
+Common uses of the table service include:
+* Storing TBs of structured data capable of serving web scale applications
+* Storing datasets that do not require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
+* Quickly querying data using a clustered index
+* Accessing data using the OData protocol filter expressions
+
+The following components make up the Azure Tables Service:
+* The account
+* A table within the account, which contains a set of entities
+* An entity within a table, as a dictionary
+
+The Azure Tables client library for Go allows you to interact with each of these components through the
+use of a dedicated client object.
+
+### Clients
+Two different clients are provided to interact with the various components of the Table Service:
+1. **`ServiceClient`** -
+    * Get and set account settings
+    * Query, create, and delete tables within the account.
+    * Get a `Client` to access a specific table using the `NewClient` method.
+2. **`Client`** -
+    * Interacts with a specific table (which need not exist yet).
+    * Create, delete, query, and upsert entities within the specified table.
+    * Create or delete the specified table itself.
+
+### Entities
+Entities are similar to rows. An entity has a **`PartitionKey`**, a **`RowKey`**, and a set of properties. A property is a name value pair, similar to a column. Every entity in a table does not need to have the same properties. Entities are returned as JSON, allowing developers to use JSON marshalling and unmarshalling techniques. Additionally, you can use the `aztables.EDMEntity` to ensure proper round-trip serialization of all properties.
+```golang
+aztables.EDMEntity{
+    Entity: aztables.Entity{
+        PartitionKey: "pencils",
+        RowKey: "id-003",
+    },
+    Properties: map[string]interface{}{
+        "Product": "Ticonderoga Pencils",
+        "Price": 5.00,
+        "Count": aztables.EDMInt64(12345678901234),
+        "ProductGUID": aztables.EDMGUID("some-guid-value"),
+        "DateReceived": aztables.EDMDateTime(time.Date{....})
+    }
+}
+```
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Table tasks, including:
+
+* [Creating a table](#creating-a-table "Creating a table")
+* [Creating entities](#creating-entities "Creating entities")
+* [Listing entities](#querying-entities "Listing entities")
+
+
+### Creating a table
+Create a table in your account and get a `Client` to perform operations on the newly created table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
+service, err := aztables.NewServiceClient("https://<myAccountName>.table.core.windows.net", cred, nil)
+handle(err)
+resp, err := service.CreateTable("myTable")
+```
+
+### Creating entities
+Create entities in the table:
+
+```golang
+cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+handle(err)
```

Comment:
Just checking, but is 'handle' defined in the readme somewhere? If not (and it's just a placeholder) should it be something like "failOnError" or similar just to tell users what it's standing in for?

Not sure what the convention is for our readmes (or if this is even important!)

Reply:
(or maybe 'panicOnError')

Reply:
It isn't defined anywhere, wanted it to read almost like "handle error here after the call" than have a specific declaration of how to actually handle the errors